lesson_2_final_03142021

orodrig5 opened the pull request:
lesson 2 03142021, version ready to use

orodrig5 committed:
final changes to lesson 2

orodrig5 committed:
added final changes, added cleaned version

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>visualization packages?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

mntrevino13 replied:
<p>Holaaaaa</p>

orodrig5 replied:
<p>yep, adding "packages"</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>-Did you mean to delete the line that reads 'plot function tells Python what to plot an dhow to plot it'?</p><p>-decades*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

orodrig5 replied:
<p>yeah I meant to delete it, for now no need to have this function explained</p><p> </p>

orodrig5 replied:
<p>Will add it back, it does make the content more clear</p><p> </p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>We can modify the ticks on the x and/or y axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>second to last sentence: keys. These can be of any type (float, int, lists, etc)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

orodrig5 replied:
<p>I am a bit confused on this comment I think it supposed to go somewhere else</p><p> </p>

orodrig5 replied:
<p>I see, its for the work below. Noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>brackets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='10'/>

orodrig5 replied:
<p>noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>let's</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='24'/>

orodrig5 replied:
<p>noted</p><p> </p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>brackets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='28'/>

orodrig5 replied:
<p>noted</p><p> </p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>brackets*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='38'/>

orodrig5 replied:
<p>noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>change smaller to 'less than or equal'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

orodrig5 replied:
<p>noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>Do you want to leave the example that was below this on the Jupyter notebook? That way they can see it for reference</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

orodrig5 replied:
<p>We can, removed it because it randomly uses numpy without much context. We can leave it for them to explore on their own</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>last comment: check*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='59'/>

orodrig5 replied:
<p>noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>Delete extra dash on line 2</p><p>BE CAREFUL.</p><p>indefinitely bacause*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='71'/>

orodrig5 replied:
<p>noted</p>

marlonseb commented on Lesson_2_Python_II/Lesson_2_clean.ipynb:
```diff
@@ -6,49 +6,34 @@
    "source": [
```
<p>Lab:</p><p>-did you want to delete the statistic staff? seems pretty easy to figure out and they can start getting used to those terms</p><p>-on the last part to the this lab, maybe say print the number of players that are taller than 73 in</p><p>-spelling: variable, for loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theLeague-AI/TheLeagueAI_main/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='80'/>

orodrig5 replied:
<p>I did delete it because we did not introduce these methods on the lesson. If I was to ask them to "perform statistical analysis" they would not know what we are referring to... what I am going to do is to add it back to the end of the lab. I will be adding details for them to know which specific metrics we are looking for.</p>

orodrig5 committed:
Implemented changes suggested by Marlon 09/03

orodrig5 committed:
Added a missing filtering example

orodrig5 committed:
Updated clean version, removed all lesson's code

orodrig5 committed:
updated the 'clean' version of lesson 2:
This version has undoes all cells outputs and fixes a couple of spelling mistakes
